CMakeLists.txt

```diff
@@ -330,56 +333,69 @@ ginkgo_modify_flags(CMAKE_CUDA_FLAGS_DEBUG)
 ginkgo_modify_flags(CMAKE_CUDA_FLAGS_RELEASE)
 ginkgo_install()
 
+set(GINKGO_TEST_INSTALL_SRC_DIR "${Ginkgo_SOURCE_DIR}/test/build_install/")
+set(GINKGO_TEST_INSTALL_BIN_DIR "${Ginkgo_BINARY_DIR}/test/build_install/")
```

Comment:
The naming here is a bit weird? `build_install` doesn't sound like what this is doing: testing the installed Ginkgo. Maybe just `install` or repeating the test with `test_install`?

Reply:
Yes, I'm also not happy with the name. `test/test_install` just felt a bit redundant. I wanted to emphasize that we test whether the build with installed Ginkgo works, because `test_subdir` doesn't clearly state that it tests the build process.